### ISSUE TITLE
Linux app won't sync while screen is locked.

### ISSUE THREAD

**Issue by nicekiwi** [meganz/MEGAsync#492]
Ubuntu 20.04 - Gnome

When the lock-screen is active, laptop plugged in, screen open, and the system is not asleep or in stand-by, background syncing stops. 

Is this intentional or a bug?

**Comment by sanjaylord** [comment]
Yes, I believe this happens now on Windows 10 too. Locking the screen appears to pause the transfer and then the files get stuck on retrying and then restart from the beginning. Previous versions didn't have this issue. 

Would like some clarification.

**Comment by pidgeon777** [comment]
It also happened to me and it is a serious bug to fix otherwise synchronization functionality is not guaranteed.
